### PR TITLE
CMS-1038 — Design review

### PR DESCRIPTION
**Pull request by Kenneth-Siu**

### CODE REVIEW

File: directory_components/templates/directory_components/cta_card.html [hunk 1]
```diff
@@ -3,6 +3,6 @@
 <div class="cta-card">
   <img src="{% firstof img_url image_url image_src img_src image %}" alt="{{ alt_text }}">
   <div class="link-container">
-    <a class="cta-link {% if with_arrow %}with-arrow{% endif %}" href="{{ url }}">{{ text|split_lines:25 }}</a>
+    <a class="cta-link {% if with_arrow %}with-arrow{% endif %}" href="{{ url }}"><span>{{ text }}</span><br/></a>
```
Comment (richtier) [inline directory_components/templates/directory_components/cta_card.html]: are the spans adding value here?

File: directory_components/templatetags/directory_components_tags.py [hunk 2]
```diff
@@ -19,13 +19,6 @@ def static_absolute(parser, token):
     return FullStaticNode.handle_token(parser, token)
 
 
-@register.filter
-def split_lines(text, length):
```
Comment (richtier) [inline directory_components/templatetags/directory_components_tags.py]: removing a feature makes this a breaking change, so please change version by a major version to communicate that (so version 8.0.0)

File: directory_components/templates/directory_components/cta_card.html (hunk 1 above)
Comment (Kenneth-Siu) [reply directory_components/templates/directory_components/cta_card.html]: Yep! The arrow (and :after in the scss) shouldn't have a blue background, but the text does, so the span is there to give only the text a blue background.

File: directory_components/templatetags/directory_components_tags.py (hunk 2 above)
Comment (Kenneth-Siu) [reply directory_components/templatetags/directory_components_tags.py]: Actually, speaking of, @richtier, any opinions on removing something like this? I know it's not used anywhere within code in uktrade (which is why I removed it), but I'm happy to keep it if that's the preference.

Comment (richtier) [reply directory_components/templatetags/directory_components_tags.py]: if it's not used then yes by all means please remove :+1:

**Pull request merged**